src/index.html

```diff
@@ -2,16 +2,146 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form method="POST" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="form__block">
+        <legend>Personal information</legend>
+        <div class="form__field">
+          <label>
+            Surname:
+            <input type="text" name="surname" autocomplete="off">
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            Name:
+            <input type="text" name="name" autocomplete="off">
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12">
```

Comment:
fix formatting everywhere

```suggestion
            <input
              type="number"
              name="age"
              min="1"
              max="100"
              value="12"
            >
```